packages/shared/src/graphql/squads.ts

```diff
@@ -287,18 +287,6 @@ export interface SquadEdgesData {
   sourceMembers: Connection<SourceMember>;
 }
 
-export const checkUserMembership = async (
-  sourceId: string,
-  memberId: string,
-): Promise<SourceMember> => {
-  const res = await request(graphqlUrl, CHECK_USER_MEMBERSHIP, {
-    sourceId,
-    memberId,
-  });
-
-  return res.member;
-};
```

Comment:
Seen this one is still there on the API, can we remove it there too or leave it in webapp for future? It's just an export so it's not adding to the bundle size..